### PR TITLE
[FEAT] Updated readme 

### LINKED ISSUE #1284: [FEAT] Update readme
Added links to these images whenever a user will click on the following images it will redirect them to that particular page for the same 

Screenshot from 2023-03-05 14-16-41

### PR DESCRIPTION
**Pull request by himanshu1221**
<h3>Closes #1284</h3>

### CODE REVIEW

**Comment by kanishkcs** [comment]
Hey @himanshu1221  You have Created PR on wrong Branch,  change the branch from master to dev branch.

**Comment by himanshu1221** [comment]
> Hey @himanshu1221 You have Created PR on wrong Branch, change the branch from master to dev branch.

i have changed the branch @kanishkcs

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -20,4 +20,6 @@ jobs:
           port: ${{ secrets.PORT }} 
           script: | 
             cd sugarizer-dev/sugarizer 
-            git pull
\ No newline at end of file
+
```
Comment (llaske) [inline .github/workflows/main.yml:23]: This file should not be a part of the PR.

Comment (himanshu1221) [reply .github/workflows/main.yml]: I have deleted this File

**Comment by himanshu1221** [comment]
hey @llaske I have made the changes i have deleted the file from the PR you can review the file and suggest to me the changes if any

**Comment by llaske** [comment]
Nope. You must remove the main.yml from your PR not from the repo.
If I merge your PR, the file will be removed.

**Comment by himanshu1221** [comment]
i'l make a seperate PR for the same i am closing this pr

**Pull request closed without merging**